gno.land/cmd/gnoland/secrets.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+
+	"github.com/gnolang/gno/tm2/pkg/commands"
+)
+
+var (
+	errInvalidDataDir    = errors.New("invalid data directory provided")
+	errInvalidSecretsKey = errors.New("invalid number of secret key arguments")
+)
+
+const (
+	defaultSecretsDir         = "./secrets"
+	defaultValidatorKeyName   = "priv_validator_key.json"
+	defaultNodeKeyName        = "node_key.json"
+	defaultValidatorStateName = "priv_validator_state.json"
+)
+
+const (
+	nodeKeyKey             = "NodeKey"
+	validatorPrivateKeyKey = "ValidatorPrivateKey"
+	validatorStateKey      = "ValidatorStateKey"
```

Comment:
```suggestion
	validatorStateKey      = "ValidatorState"
```

Reply:
Renamed, got left in the copypasta:

6965f12